diff --git a/src/next-transpile-modules.d.ts b/src/next-transpile-modules.d.ts
index 9fc7d0a28176b649fcadb6d1bd6859c72bca9595..c0ba96230af8dcc669f06af99da6d5226800c7b8 100644
--- a/src/next-transpile-modules.d.ts
+++ b/src/next-transpile-modules.d.ts
@@ -8,4 +8,4 @@ declare function withTmInitializer(modules?: string[], options?: {
     resolveSymlinks?: boolean;
     debug?: boolean;
     __unstable_matcher?: (path: string) => boolean;
-}): (nextConfig?: {}) => {};
+}): <T>(nextConfig?: T) => T;
